extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import type * as azdataType from 'azdata';
+import * as azdataType from 'azdata';
```

Comment:
in that case, how do i call the azdata.designers.openTableDesigner?

I am getting the error "'azdataType' cannot be used as a value because it was imported using 'import type'.ts(1361)
"

Reply:
You call utils.getAzdataApi().designers.openTableDesigner

e.g. https://github.com/Microsoft/azuredatastudio/blob/b6047ad87db1d471bc3813d5fc5ce7ab7788699b/extensions/sql-database-projects/src/dialogs/addDatabaseReferenceDialog.ts#L55

You need to do this because the imports are ran as soon as the module is loaded (which happens on extension activation generally), but in VS Code the azdata module doesn't exist so it'll throw if you reference it directly.